pkg/build/controller/build/defaults/defaults.go

```diff
@@ -100,6 +103,14 @@ func (b BuildDefaults) applyPodDefaults(pod *v1.Pod) {
 			}
 		}
 	}
+
+	// Apply default Tolerations
+	if len(pod.Spec.Tolerations) == 0 && len(b.config.Tolerations) != 0 {
```

Comment:
I think we decided you should append these even if pod.Spec.Tolerations already contains some tolerations?